operator/src/network/ceramic.rs

```diff
@@ -327,6 +322,17 @@ pub fn stateful_set_spec(ns: &str, bundle: &CeramicBundle<'_>) -> StatefulSetSpe
         },
     ];
 
+    // Only specify the pubsub topic if the network was "inmemory" or "local"
```

Comment:
Can you update this comment for why this is important? Does js-ceramic break if we specify it for some other reason?

Reply:
Yes, js-ceramic won't come up if the pubsub topic is specified for any network type but inmemory/local.